Pipfile

```diff
@@ -21,10 +17,9 @@ python-jose = "==1.3.2"
 redis = "==2.10.6"
 requests = "*"
```

Comment:
Could pin gunicorn and requests?
`gunicorn = "==19.8.0"`
`requests = "==2.18.4"`